requirements.txt

```diff
@@ -1,3 +1,7 @@
+# This file is redundant with setup.cfg;
```

Comment:
If it's redundant but it must exist, what's the advantage of duplicating the dependencies in setup.cfg? This duplication seems brittle, since we have to manually ensure the files are never out of sync.

Reply:
In future GitHub features hopefully this file won't be necessary